otherlibs/dune-rpc/private/where.ml

```diff
@@ -58,6 +58,8 @@ let rpc_socket_relative_to_build_dir = ".rpc/dune"
 
 let _DUNE_RPC = "DUNE_RPC"
```

Comment:
Why do we need this? Isn't `env_var` enough?

Reply:
(I guess the answer is "to make it more readable", in which case it may be worth adding a comment.)

Reply:
It just helps to catch typos. E.g. if I was to right "DUNE_RPX" somewhere, the compiler wouldn't catch it.

Reply:
Sorry, I think I'm still missing something. We have two names for the same string: `_DUNE_RPC` and `env_var`, and either of them can be used to catch typos. Why do we need both?

Reply:
My brain short circuited. One of the variables is gone.